diff -urNp botman-1.2.16/src/lib/Makefile.mk botman-1.2.16-new/src/lib/Makefile.mk
--- botman-1.2.16/src/lib/Makefile.mk	2015-12-04 15:06:17.000000000 +0100
+++ botman-1.2.16-new/src/lib/Makefile.mk	2016-03-01 10:23:59.382867061 +0100
@@ -25,7 +25,7 @@ src_lib_libgotham_la_CFLAGS =           
    $(GOTHAM_CFLAGS)
 src_lib_libgotham_la_CPPFLAGS =                                                \
    $(AM_CPPFLAGS) -DEFL_GOTHAM_BUILD                                           \
-   -DGOTHAM_MODULE_PATH=\"$(libdir)/gotham/$(MODULE_ARCH)\"
+   -DGOTHAM_MODULE_PATH=\"$(libdir)/gotham\"
 src_lib_libgotham_la_LDFLAGS = $(GOTHAM_LIBS)                                  \
    version-info @version_info@ -no-undefined
 src_lib_libgotham_la_LIBADD =                                                  \
diff -urNp botman-1.2.16/src/modules/Makefile.mk botman-1.2.16-new/src/modules/Makefile.mk
--- botman-1.2.16/src/modules/Makefile.mk	2015-12-04 15:06:17.000000000 +0100
+++ botman-1.2.16-new/src/modules/Makefile.mk	2016-02-29 11:35:02.482631280 +0100
@@ -1,7 +1,7 @@
 MAINTAINERCLEANFILES += \
 src/lib/modules/seen/*.gc{no,da}
 
-moddir = $(libdir)/gotham/$(MODULE_ARCH)
+moddir = $(libdir)/gotham
 mod_LTLIBRARIES = \
    src/modules/access.la \
    src/modules/help.la \
